<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPage</class>
 <widget class="QWidget" name="MainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>691</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="mainLayout" stretch="3,1">
   <item>
    <layout class="QVBoxLayout" name="leftLayout" stretch="1,6">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_bird_risk_status">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>30</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string>background-color: #fffac8; color: #b8860b; font-weight: bold;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>주의</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_rwy1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>RWY 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_rwy2">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>30</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>RWY 2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_rwy2_status">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>30</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string>background-color: #ccffcc; color: #228b22; font-weight: bold;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>안전</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_rwy1_status">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>30</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string>background-color: #ffcccc; color: #b22222; font-weight: bold;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>위험</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_bird_risk">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>30</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>조류 위험도</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="map_cctv_stack">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>400</height>
        </size>
       </property>
       <widget class="QFrame" name="map_overlay_frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <layout class="QGridLayout" name="overlayStack">
         <item row="0" column="0">
          <widget class="QLabel" name="label_map">
           <property name="minimumSize">
            <size>
             <width>600</width>
             <height>400</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QWidget" name="marker_overlay_placeholder" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_cctv_1">
        <property name="text">
         <string>CCTV 1 영상</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_cctv_2">
        <property name="text">
         <string>CCTV 2 영상</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_cctv_3">
        <property name="text">
         <string>CCTV 3 영상</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="rightLayout" stretch="1,1,11">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="topButtonLayout">
       <item>
        <widget class="QPushButton" name="btn_show_map">
         <property name="text">
          <string>지도 보기</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="cctvLayout" stretch="3,1">
       <item>
        <widget class="QComboBox" name="combo_cctv">
         <item>
          <property name="text">
           <string>CCTV 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CCTV 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CCTV 3</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_show_cctv">
         <property name="text">
          <string>보기</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="object_area">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="object_table_page">
        <layout class="QVBoxLayout" name="objectTableLayout">
         <item>
          <widget class="QTableWidget" name="table_object_list">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>400</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <column>
            <property name="text">
             <string>ID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>위치</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>종류</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_detail">
           <property name="text">
            <string>상세 보기</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="detail_page">
        <layout class="QVBoxLayout" name="detailLayout"/>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
